ui/tests/acceptance/enterprise-replication-test.js

```diff
@@ -12,9 +12,7 @@ const disableReplication = (type, assert) => {
       if (assert) {
         andThen(() => {
           assert.equal(currentURL(), `/vault/replication/${type}`, 'redirects to the replication page');
-          assert.equal(
-            // TODO better test selectors for flash messages
-            find('[data-test-flash-message-body]:contains(This cluster is having)').text().trim(),
+          assert.dom('[data-test-flash-message-body]:contains(This cluster is having)').hasText(
```

Comment:
`:contains` from jQuery doesn't have an easy equivalent, and I didn't see an alternative in qunit-dom - but! I think if you just want to comment out the assertions that use this (I think there's just a couple), we could merge and we can add them back in with the `ember-cli-page-object` we have for flash-messages instead.